pyls/workspace.py

```diff
@@ -29,6 +81,17 @@ def __init__(self, root_uri, lang_server=None):
         self._docs = {}
         self._lang_server = lang_server
 
+        # Whilst incubating, keep private
+        self.__rope = Project(self._root_path)
+        self.__rope.prefs.set('extension_modules', self.PRELOADED_MODULES)
+        # jedi.api.preload_module(*self.PRELOADED_MODULES)
```

Comment:
What did you discover / decide with this?

Reply:
Rope autocompletion of preloaded modules is much faster than Jedi, also I found that by using the Jedi preloading functions, the server gets blocked.